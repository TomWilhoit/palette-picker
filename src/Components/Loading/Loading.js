import React from 'react'

export const Loading = () => <h2>Loading Projects and Palettes</h2>