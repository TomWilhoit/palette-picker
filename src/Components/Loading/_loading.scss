h2 {
  color: rgb(67, 93, 148);
  font-size: 99%;
  margin: 0;
  padding: 0;
}