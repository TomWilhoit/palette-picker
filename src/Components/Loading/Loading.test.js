import React from "react";
import { shallow } from "enzyme";
import { Loading } from "./Loading";

describe("Loading", () => {
  it("should match snapshot", () => {

  });
});