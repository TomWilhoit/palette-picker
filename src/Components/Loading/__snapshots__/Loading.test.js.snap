// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot with all data passed in 1`] = `
<h2>
  Loading Saved 
  <br />
   Projects and Palettes...
</h2>
`;
